packages/dynamite/dynamite_runtime/lib/src/http_client.dart

```diff
@@ -134,7 +135,8 @@ class DynamiteClient {
     final Map<String, String> headers,
     final Uint8List? body,
   ) async {
-    final uri = Uri.parse('$baseURL$path');
+    final uri = (UriBuilder.fromUri(baseURL)..path = '${baseURL.path}$path').build();
```

Comment:
Why not just use `Uri.replace`? I can see how this is nicer to work with, but I don't see it as necessary (at least in this case).

Reply:
AFAICT this will lead to side effects mutating the baseURL object.

Or am I wrong :thinking:

Reply:
No you get a new Uri returned